plotRisk.py

```diff
@@ -36,33 +42,33 @@ def plotRisk(self, attacks): # goes through enemies potential attacks and damage
                         healthDamage = attack.pop(0)
                         
                         if self.mechanical <= mechDamage:
-                                att = f"{attackName} {risks[0]}"
+                                att = f"{attackName} {risk['high']}"
                                 print(att) #will "stun" the character 
                                 continue  
                         totalDamage = 0
                         totalDamage += magicDamage - self.magicDefence if self.magicDefence < magicDamage else 0
```

Comment:
self.magicDefence should be self.magicDefense. Spelling error for every instance of "Defence."